# HG changeset patch
# User Sean Farley <sean@mcs.anl.gov>
# Date 1332269670 18000
#      Tue Mar 20 13:54:30 2012 -0500
# Node ID b2df50e909699a8bdea4feaa083a4d16343a2796
# Parent  00f36f7c4dddc0637f81015c1213bc4a3b6c9e93
make: add a cflags variable to pass to cmake

diff -r 00f36f7c4ddd -r b2df50e90969 Makefile
--- a/Makefile	Tue Mar 20 13:54:31 2012 -0500
+++ b/Makefile	Tue Mar 20 13:54:30 2012 -0500
@@ -7,10 +7,11 @@ gprof           = not-set
 openmp          = not-set
 prefix          = not-set
 gklib_path      = not-set
 shared          = not-set
 cc              = not-set
+cflags          = not-set
 longindex       = not-set
 doubleprecision = not-set
 
 
 # Basically proxies everything to the builddir cmake.
@@ -51,10 +52,13 @@ ifneq ($(shared), not-set)
     CONFIG_FLAGS += -DSHARED=1
 endif
 ifneq ($(cc), not-set)
     CONFIG_FLAGS += -DCMAKE_C_COMPILER=$(cc)
 endif
+ifneq ($(cflags), not-set)
+	CONFIG_FLAGS += -DCMAKE_C_FLAGS:STRING="$(cflags)"
+endif
 ifneq ($(longindex), not-set)
     CONFIG_FLAGS += -DMETIS_USE_LONGINDEX=$(longindex)
 endif
 ifneq ($(doubleprecision), not-set)
     CONFIG_FLAGS += -DMETIS_USE_DOUBLEPRECISION=$(doubleprecision)
